.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: csharp
+mono: 4.2.1
+
+sudo: false
+
+cache:
+  directories:
+  - OpenRA/thirdparty/download
+
+addons:
+  apt:
+    packages:
+    - lua5.1
+
+script:
+ - cd OpenRA.Mods.RA2
+ - xbuild
+ - cd ../../..
+ - mkdir -p ~/.openra/mods
+ - mv OpenRA/ra2 ~/.openra/mods/ra2
+ - git clone https://github.com/OpenRA/OpenRA.git
+ - cd OpenRA
+ - travis_retry make all-dependencies
+ - make all
+ - mono --debug OpenRA.Utility.exe ra2 --check-yaml
+
+branches:
+ only:
+   - master
+
+# Notify developers when build passed/failed.
+notifications:
+  irc:
+    template:
+      - "%{repository}#%{build_number} %{commit} %{author}: %{message} %{build_url}"
+    channels:
+      - "irc.freenode.net#openra"
```

Comment:
We are part of the guys in OpenRA ;) and the people working on this are already too far away from the core team anyway. If at all, this improves communication.